### PR TITLE
Oh my....

### PR DESCRIPTION
**Pull request by pieromqwerty**
- Fixed teams folder punctuation
- Grammar and spelling fixes for tutorials folder.
- Piero can no longer look at the tutorials without feeling exhausted.
- Made general grammar more professional.

### CODE REVIEW

File: teams/roles.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Different Roles 
+
+
+Member -> This is anyone that gets invited is immediately given this role. They are essentially a student.
```
Comment (jeremypress) [inline teams/roles.md:4]: while you're here. Can you fix up this first sentence? -> Anyone that is invited to the team starts with this role.

File: teams/invitations.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
-## Inviting people to your team
+## Inviting People to your Team
 
 Follow the steps during team creation to invite people with their repl.it username or email addresses.
 
-If you've already created a team, go to the team dashboard and under "Teammates" click "Add more."
+If you have already created a team, go to the team dashboard and under "Teammates" click "Add more."
```
Comment (jeremypress) [inline teams/invitations.md:5]: nothing wrong with conjunctions imo, we have a professional but casual and playful tone in our comms.

File: teams/invitations.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
-## Inviting people to your team
+## Inviting People to your Team
```
Comment (jeremypress) [inline teams/invitations.md:1]: can you fix this for privacy invite links too?

**Review by jeremypress — CHANGES_REQUESTED** [review]
Thanks for touching these up! Just a couple small tweaks and we can :boat:

File: teams/invitations.md (hunk 2 above)
Comment (pieromqwerty) [reply teams/invitations.md:5]: I'll go ahead and change those.

File: teams/invitations.md (hunk 3 above)
Comment (pieromqwerty) [reply teams/invitations.md:1]: Sure thing!

**Comment by pieromqwerty** [comment]
@jeremypress Should be all done!

**Pull request closed without merging**